tools/update-tester/src/main.rs

```diff
@@ -149,6 +150,8 @@ fn main() {
                     .long("database")
                     .takes_value(true)
             )
+
+            .arg(Arg::new("ROOT_DIR").takes_value(true))
```

Comment:
It would be better to require the current directory be the root of the project rather than making the caller pass that directory as argument.

`open(extension/toolkit.control).unwrap()`

Reply:
Should I make that a requirement for all 3 commands that use root_dir?
I could also change that line to           ```.arg(Arg::new("ROOT_DIR").takes_value(true).default_value("."))``` and then we could keep the flexibility of calling it from other dirs and wouldn't have to update any scripts that call it.

Reply:
Updated release automation plan to include this in future work.